# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/pointerevents/pointerlock/pointerevent_pointerrawupdate_in_pointerlock.https.html.ini
# Commit: 5718f7f3067e
# Full Hash: 5718f7f3067eff42e097a6013acd82fbe6aa299a
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-05-15 22:58:00
# Regressor Bug: 1550462
# File Overlap Count: 1
# Description:
#   Bug 1550462 - part 2: Make `PresShell::HandleEvent` dispatch preceding `pointerrawupdate` event r=smaug,dom-core,edgar
#   
#   This patch tries to dispatch `ePointerRawUpdate` with
#   `PresShell::EventHandler::DispatchPrecedingPointerEvent` as same as usual
#   pointer events.
# ==============================================================================

diff -r aae164cff6aa -r 5718f7f3067e testing/web-platform/meta/pointerevents/pointerlock/pointerevent_pointerrawupdate_in_pointerlock.https.html.ini
--- a/testing/web-platform/meta/pointerevents/pointerlock/pointerevent_pointerrawupdate_in_pointerlock.https.html.ini	Thu May 15 03:20:28 2025 +0000
+++ b/testing/web-platform/meta/pointerevents/pointerlock/pointerevent_pointerrawupdate_in_pointerlock.https.html.ini	Thu May 15 03:20:29 2025 +0000
@@ -1,4 +1,5 @@
 [pointerevent_pointerrawupdate_in_pointerlock.https.html]
+  max-asserts: 1 # in PointerEventHandler::DispatchPointerFromMouseOrTouch
   expected:
     if (os == "win") and debug and (processor == "x86_64"): [OK, TIMEOUT]
     if (os == "win") and debug and (processor == "x86"): TIMEOUT